addons/web_editor/static/src/components/history_dialog/history_dialog.js

```diff
@@ -37,6 +38,9 @@ class HistoryDialog extends Component {
     }
 
     async init() {
+        const isDarkMode = cookie.get("color_scheme") === "dark";
+        document.documentElement.style.setProperty('--border-color', `var(--border-color-${isDarkMode ? "dark" : "normal"})`);
```

Comment:
that's what i wanted to do first, i searched in the code and didn't find an example. so i decided to do it in js instead of blocking the task. keep me updated to adapt the changes

Reply:
I found another solution to do it:

```diff
diff --git a/addons/web_editor/__manifest__.py b/addons/web_editor/__manifest__.py
index 674bd9c7395c..409c4a491404 100644
--- a/addons/web_editor/__manifest__.py
+++ b/addons/web_editor/__manifest__.py
@@ -209,6 +209,7 @@ Odoo Web Editor widget.
             'web_editor/static/src/js/backend/**/*',
             'web_editor/static/src/xml/backend.xml',
             'web_editor/static/src/components/history_dialog/**/*',
+            ('remove', 'web_editor/static/src/components/history_dialog/history_dialog.dark.scss'),
         ],
         "web.assets_web_dark": [
             'web_editor/static/src/scss/odoo-editor/powerbox.dark.scss',
@@ -216,6 +217,7 @@ Odoo Web Editor widget.
             'web_editor/static/src/scss/odoo-editor/tableui.dark.scss',
             'web_editor/static/src/scss/wysiwyg.dark.scss',
             'web_editor/static/src/scss/web_editor.common.dark.scss',
+            'web_editor/static/src/components/history_dialog/history_dialog.dark.scss',
         ],
         'web.assets_frontend_minimal': [
             'web_editor/static/src/js/frontend/loader_loading.js',
diff --git a/addons/web_editor/static/src/components/history_dialog/history_dialog.js b/addons/web_editor/static/src/components/history_dialog/history_dialog.js
index 401749566bb1..10c1690930b9 100644
--- a/addons/web_editor/static/src/components/history_dialog/history_dialog.js
+++ b/addons/web_editor/static/src/components/history_dialog/history_dialog.js
@@ -6,7 +6,6 @@ import { useService } from '@web/core/utils/hooks';
 import { memoize } from '@web/core/utils/functions';
 import { Component, onMounted, useState, markup } from '@odoo/owl';
 import { _t } from '@web/core/l10n/translation';
-import { cookie } from "@web/core/browser/cookie";

 const { DateTime } = luxon;

@@ -38,8 +37,6 @@ class HistoryDialog extends Component {
     }

     async init() {
-        const isDarkMode = cookie.get("color_scheme") === "dark";
-        document.documentElement.style.setProperty('--border-color', `var(--border-color-${isDarkMode ? "dark" : "normal"})`);
         document.getElementsByClassName("modal-content")[0].style.height = "100%";
         this.state.revisionsData = this.props.historyMetadata;
         await this.updateCurrentRevision(this.props.historyMetadata[0]['revision_id']);
diff --git a/addons/web_editor/static/src/components/history_dialog/history_dialog.scss b/addons/web_editor/static/src/components/history_dialog/history_dialog.scss
index d04ed62e2168..c25544ec47e3 100644
--- a/addons/web_editor/static/src/components/history_dialog/history_dialog.scss
+++ b/addons/web_editor/static/src/components/history_dialog/history_dialog.scss
@@ -1,10 +1,6 @@
-:root {
-  --border-color-normal: #ddd;
-  --border-color-dark: #3C3E4B;
-}
-
 .html-history-dialog {
     .history-container {
+        --border-color: #ddd;
         margin-left: 240px;
         >div {
             padding: 10px 12px;
```